app/controllers/mobile_dashboard_controller.rb

```diff
@@ -0,0 +1,26 @@
+class MobileDashboardController < ApplicationController
+  #before_action :authenticate_user!
+  #before_action :authenticate_control!
```

Comment:
Probably need to uncomment authenticate_user, but I suspect we don't need authenticate control?